docs/comparison.rst

```diff
@@ -0,0 +1,83 @@
+================================
+
+pyuvsim is designed to calculate visibilities from point sources and a given primary beam model. It does this through direct calculation of the full-polarization Radio Interferometer Measurement Equation (RIME), fully parallelized across a set of MPI processes. Its development has progressed with the goal of providing a tool that can be used reliably and flexibly, but most of all it has focused on avoiding effects that confound foreground modeling efforts in 21cm cosmology.
+
+As such, it has relatively poor performance compared with several other simulators. Deciding which tool to use will ultimately depend on what factors are important to the user. We recommend pyuvsim as a reference tool to verify the results of other simulators, and for producing limited datasets for comparison with observations (particularly on long baselines). Users will need access to a lot of memory and a large number of processor cores to take advantage of pyuvsim's accuracy and precision, ideally on a dedicated high-performance computing cluster.
+
+On this page, we compare pyuvsim's features and performance to several other visibility simulators.
+
+Precision Radio Interferometry Simulator (PRISim)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A python package for generating full-sky simulations.
+
+Advantages:
+
+1. Faster due to vectorized calculations. 
+2. Well-tested through years of use.
+3. Support for a range of sky models including point sources, diffuse emission, and spectral cubes.
+4. Support for tracking as well as transit telescopes.
+5. Parallelized over baselines, frequencies, or sky model/catalog.
+6. Has been tested using imaging with CASA.
+
+Disadvantages:
+
+1. MPI overhead is large if time axis in simulation is long.
+2. Supports only one polarization at a time.
+3. Does not support non-identical antenna patterns.
+
+Fast Holographic Deconvolution (FHD)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+FHD is a data calibration and modeling framework written in IDL by researchers at the University of Washington for analyzing data from the Murchison Widefield Array (MWA) in Western Australia. It calculates sky models by convolving a high-resolution psf with the exact DFT of point sources. This is approach is more efficient when working with wide primary beams, which will be narrower in Fourier space and thus limit the size of the convolution to be performed.
+
+Advantages:
+
+1. Fast.
+2. Well-tested through years of use.
+3. Support for diffuse foreground models.
+4. Support for discrete pointings.
+5. Uses IDL's native parallelization efficiently.
+
+Disadvantages:
+
+1. Written in the proprietary IDL language.
+2. Designed for the MWA, and has limited support for other instruments.
+3. The discrete convolution in Fourier Space can introduce aliasing and ringing artifacts at a level relevant to 21cm cosmology.
+4. Measurement equation formalism relies on the flat-sky approximation, limiting curved sky effects.
```

Comment:
I need to talk to Ian about this, I know there are some curved sky effects handled in FHD.

Reply:
Yeah, 4 is mostly wrong and I'd just delete. We wrote FHD _because_ we needed the curved sky effects. This is literally why we started writing it. So FHD has all the curved sky effects in, with the small caveat that it cannot model the sky more than 90° away from the phase center (uv integration approaches fall down in this limit). 

2 is also strange. It certainly has been tested almost exclusively on MWA data, but we have made images of PAPER and HERA data and simulated all sorts of other arrays. Importing beams has been a sticking point, but this should be resolved in a matter of weeks.

Reply:
Trying to think of disadvantages to add. 

One is that high resolution performance is largely untested (but this might be doubly true of the others). 

Another is that it memory scales with the size of the uv-plane. This helps it be very fast for compact 21 cm cosmology arrays, but for higher resolution arrays with sparse uv coverage the memory scaling can be poor.

Reply:
I'll remove (4). I was thinking mainly about how the visibility equation can be written as a 2d Fourier transform only in a flat sky approximation, but of course FHD is sophisticated enough to handle the curved sky effects.

Reply:
Yeah, FHD does this simply by working in ortho-slant image coordinates+w-projection snapshots, which is a fairly standard fix for this issue.